File: C/problem001.c
/* Find the sum of all the multiples of 3 or 5 1000 below */

#include <stdio.h>
int sumofmultiple(int multinumber,int maxnumber)
{
    int n = (maxnumber-1) / multinumber;
    return n * (n + 1) / 2 * multinumber;
}
int main()
{
    int sum = sumofmultiple(3, 1000) + sumofmultiple(5, 1000) - sumofmultiple(15, 1000);
    printf("%d", sum);
    return 0;
}

File: Python/problem001.py
# Find the sum of all the multiples of 3 or 5 1000 below 

def total(multi_number:int,last_number:int):

    n = (last_number-1)//multi_number

    return multi_number*n*(n+1)/2


    
result = total(3,1000) + total(5,1000) - total(15,1000)
print(result)